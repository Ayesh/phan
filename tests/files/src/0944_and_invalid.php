<?php

/**
 * @param non-empty-list $a not a valid class name
 * @param phan-intersection-type $b without template args, this is mixed
 */
function test($a, $b) {
    '@phan-debug-var $a, $b';
}
