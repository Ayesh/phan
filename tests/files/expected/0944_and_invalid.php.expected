%s:7 PhanDebugAnnotation @phan-debug-var requested for variable $a - it has union type non-empty-list<mixed>
%s:7 PhanDebugAnnotation @phan-debug-var requested for variable $b - it has union type mixed
